app/main.py

```diff
@@ -14,19 +15,48 @@
     "7e8f0ada0a03cbee48a0883d549967647b3fca6efeb0a149242f19e4b68d53d6",
     "e5f3ff26aa8075ce7513552a9af1882b4fbc2a47a3525000f6eb887ab9622207",
 ]
+RANGES = [(x, x + 10000000) for x in range(100000000) if x % 10000000 == 0]
```

Comment:
What about this? It looks much faster: 
```suggestion
EIGHT_DIGITS_NUMBER = 10000000
RANGES = [(i*EIGHT_DIGITS_NUMBER, (i+1)*EIGHT_DIGITS_NUMBER) for i in range(10)]
```
We have only 10 iterations in this case. So, it should add some speed, but doing the same thing. 

<img width="732" alt="image" src="https://github.com/mate-academy/py-brute-force-password/assets/37514514/b26904ab-0ee3-48c1-87db-80d11a5e7240">

Reply:
<img width="800" alt="Screenshot 2023-10-03 141746" src="https://github.com/mate-academy/py-brute-force-password/assets/53839429/9c04bf9d-1490-4a32-86c1-42b6161d38a7">

Reply:
It is faster, but the variant without loops is still the fastest))